index.js

```diff
@@ -70,13 +109,15 @@ app.post("/users/:id/movies/:id", (req, res) => {
   );
   });
 //removes movie from the list of favorites of a user
-app.delete("/users/:id/movies/:movie_id", (req, res) => {
-  Users.findOneAndRemove({FavoriteMovies: req.params.FavoriteMovies})
```

Comment:
I this case the method `findOneAndRemove` would be removing/deleted a **user**, but what we want instead is to update the user so a movie gets removed from their list of favorites. This is why the method `Users.findOneAndUpdate` should be used instead.